packages/components/src/components/as-switch/as-switch.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../../styles/src/core/_dev';
+
+as-switch {
+  display: inline-block;
```

Comment:
Are these styles copied from styles folder? Why don't we reference them to avoid changing the code twice?

Reply:
They are different :(

Reply:
anyway we should remove the switch element

Reply:
I do think so too! Let's do it.